app/src/editor/canvas/components/ModuleSearch.jsx

```diff
@@ -114,14 +152,29 @@ export class ModuleSearch extends React.PureComponent<Props, State> {
 
     componentDidMount() {
         window.addEventListener('keydown', this.onKeyDown)
+        this.addOrRemoveDropListener(true)
         this.load()
     }
 
     componentWillUnmount() {
         this.unmounted = true
+        this.addOrRemoveDropListener(false)
         window.removeEventListener('keydown', this.onKeyDown)
     }
 
+    addOrRemoveDropListener= (add: boolean) => {
+        const canvasElement = document.querySelector(`.${CanvasStyles.Modules}`)
```

Comment:
yeah this this could be passed but yeah it'd have to go up out of Canvas then back down into this, blegh. Ok I guess this is fine for now.